security/ldap-config-handler/src/main/java/org/codice/ddf/admin/security/ldap/test/LdapTestingCommons.java

```diff
@@ -96,6 +97,9 @@ public LdapConnectionAttempt getLdapConnection(LdapConfiguration ldapConfigurati
     }
 
     public LdapConnectionAttempt bindUserToLdapConnection(LdapConfiguration ldapConfiguration) {
+        // This ConnectionAttempt intentionally not closed as its internal connection is bound
```

Comment:
Did you come up with any ideas how we can avoid passing a connection around? I'd be more than willing to try something different when this is ported to graphql. It'd be great if we could concentrate all of this into one place so everyone doesn't have to keep doing the exception handling and logging.